### PR TITLE
Update OpenVINO supported releases

### PR DESCRIPTION
**Pull request by pamolloy**
There were some other references to OpenVINO 2020.1 in the docs that I'll update to point to this section of the FAQ.

### CODE REVIEW

**Comment by szabi-luxonis** [comment]
This is valid only for gen2.

**Comment by pamolloy** [comment]
@szabi-luxonis so Gen 1 only supports 2020.1? I can update it to make that distinction.

**Comment by szabi-luxonis** [comment]
> @szabi-luxonis so Gen 1 only supports 2020.1? I can update it to make that distinction.

Yes.

File: source/pages/faq.rst [hunk 1]
```diff
@@ -305,7 +305,10 @@ See :ref:`here <Multiple DepthAI per Host>` for instructions on how to do so.
 Is DepthAI OpenVINO Compatible?
 ###############################
 
-Yes.  As of this writing, DepthAI is fully compatible with OpenVINO 2020.1.  We are in the process of upgrading to have compatibility with newer OpenVINO versions.
+Yes.  DepthAI Gen 1 is fully compatible with OpenVINO 2020.1.
+
+.. note::
+   `DepthAI Gen 2 <https://docs.luxonis.com/projects/api/en/gen2_develop/>`__ supports 2020.1, 2020.2, 2020.3, 2020.4 and 2021.1.  We do our best to support new OpenVINO releases as soon as they are made available.
```
Comment (VanDavv) [inline source/pages/faq.rst:311]: after a directive (`.. note::` in this case) there should be an extra empty line

**Comment by VanDavv** [comment]
LGTM

**Review by VanDavv — APPROVED** [review]

**Pull request merged**